engines/python/src/test/resources/oom/serving.properties

```diff
@@ -0,0 +1,4 @@
+option.pythonExecutable=python3
```

Comment:
this file is unnecessary, we can remove it

Reply:
updated.